firelyserver/releasenotes/upgrade.rst

```diff
@@ -12,8 +12,8 @@ This page describes the general process for each situation. Please refer to the
 
 .. _upgrade_server: 
 
-Upgrading Firely Server FHIR Server
---------------------------
+Upgrading Firely Server
```

Comment:
Following these steps for 3.x to 4.x will not work, something should be mentioned here about the renames one has to do.